v2/internal/attacktechniques/aws/credential-access/ec2-get-password-data/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.71.0"
+      version = "~> 3.0"
```

Comment:
(Note: In terraform's terms this means "Use 3.x, whatever is the most recent". Currently it will use `3.75.1`